samples/javascript/videoshare/README.md

```diff
@@ -0,0 +1,62 @@
+# Video Share: Real time binary pubsub sample
+
+This is a sample project to demonstrate how to use protobuf subprotocol to achieve a binary based client pubsub:
+
+This application is based on the following technologies:
+
+* For frontend: HTML5/javascript, bootstrap and vue.js
+* For backend: node.js + express.js
+* For realtime communication: Azure Web PubSub with protobuf subprotocol
+
```

Comment:
How about adding an screenshot of the demo to show the goal and also attract reader to run the sample?